### PR TITLE
Show package_groups without patternType

### PR DESCRIPTION
**Pull request by DavidKang**
Now we show all `package_groups` with type 'image' independently of the pattenType.

#### Showing patternType

screenshot from 2017-09-21 13-41-38

### CODE REVIEW

File: src/api/app/models/kiwi/image.rb [hunk 1]
```diff
@@ -141,7 +141,7 @@ def outdated?
   end
 
   def default_package_group
-    package_groups.find_or_create_by(kiwi_type: :image)
```
Comment (bgeuken) [inline src/api/app/models/kiwi/image.rb]: It's not causing any problem that don't create this if it's missing?

Comment (DavidKang) [reply src/api/app/models/kiwi/image.rb]: uhhmm, good point!

**Comment by ChrisBr** [comment]
Hmm, then we have the problem again that the user doesn't know to which group he should add the package.

I think the big problem what we have is the import: We have templates with package group with type='onlyRequired' and we have templates without it. So I think we should set the ``default_package_group`` to what is provided when we import the image.

**Comment by ChrisBr** [comment]
Just as an idea and quick and dirty:

```
def default_package_group
  if package_groups.where(kiwi_type: :image).count > 2
    # add an error
  end
  package_groups.first || package_groups.find_or_create_by(kiwi_type: :image, pattern_type: 'onlyRequired')
end
```

If we have more than two package groups we have an error otherwise we set the default to what is supplied, either ``type=image patter_type=OnlyRequired`` or ``type=image``, if there is nothing supplied we use as default ``type=image patter_type=OnlyRequired``.
Does that make sense?

**Comment by DavidKang** [comment]
@ChrisBr, make sense, but in this case, I prefer to make the validation in the `build_from_xml` method and if fails send him to the package_view_file to fix the package_group.

**Comment by ChrisBr** [comment]
@DavidKang yep, agreed!

**Comment by DavidKang** [comment]
Now the cassettes are update

**Comment by DavidKang** [comment]
@ChrisBr, could you re-review it again 😸

**Comment by ChrisBr** [comment]
LGTM, thanks @DavidKang :heart:

**Pull request merged**